client/app/data/plugins/deck.ts

```diff
@@ -110,12 +111,16 @@ export const deck = t.router({
 			const deckIndex = ship.decks.findIndex(
 				(deck) => deck.name === input.deckId,
 			);
-			if ("newName" in input) {
+			if ("generateName" in input) {
 				deck.name = generateIncrementedName(
-					input.newName,
+					input.generateName,
 					ship.decks.map((deck) => deck.name),
 				);
 			}
```

Comment:
I don't think this is used anywhere in the UI now, so I don't think we need to keep this here. You can remove the whole "generateName" case.